### PR TITLE
Updatehttps://github.com/aws/aws-iot-device-sdk-java README.md

### PR DESCRIPTION
**Pull request by mannesuresh**
https://github.com/aws/aws-iot-device-sdk-java
https://github.com/aws/aws-iot-device-sdk-java
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by mannesuresh** [comment]
This product very good marketing

File: README.md [hunk 1]
```diff
@@ -217,7 +217,8 @@ To subscribe to a topic:
 public class MyTopic extends AWSIotTopic {
     public MyTopic(String topic, AWSIotQos qos) {
         super(topic, qos);
-    }
+    }on a  nice good project
```
Comment (graebm) [inline README.md]: too many spaces, should be
```}on a  nice good project```

File: README.md [hunk 2]
```diff
@@ -401,94 +402,4 @@ dependencies.
 
 * Publish/Subscribe sample:
 This sample consists of two publishers publishing one message per second to a 
-topic. One subscriber subscribing to the same topic receives and prints the 
-messages. 
-
-* Shadow sample:
-This sample consists of a simple demo of the simplified shadow access 
-model. The device contains two attributes: window state and room temperature.
```
Comment (graebm) [inline README.md]: forgot to delete the copyright

**Comment by graebm** [comment]
closed weird garbage PR

**Pull request closed without merging**